cub/cub/cmake/cub-config-version.cmake

```diff
@@ -1,17 +1,10 @@
 # Parse version information from version.cuh:
 include("${CMAKE_CURRENT_LIST_DIR}/cub-header-search.cmake")
```

Comment:
@allisonvacanti I'd assumed this `include()` was only relevant when parsing the version information from the header files and I could delete it now, but things break if I do that.

Is that expected?

Reply:
Those scripts detect the project include dir, which is re-used to setup the CMake targets after pulling out the version info.

Reply:
As long as we're updating the headers, it should be fine to leave the parsing logic in place in these files.

Reply:
Call me superstitious, but it just makes me itchy to have the cmake files parse out the version from the headers. I'd rather have it hardcoded and updated via the `update_version` script, so that way the script tells us everywhere that version information is located.